include/odp/api/spec/packet.h

```diff
@@ -2140,6 +2140,32 @@ uint32_t odp_packet_payload_offset(odp_packet_t pkt);
  */
 int odp_packet_payload_offset_set(odp_packet_t pkt, uint32_t offset);
 
+/**
+ * Enable or disable Tx packet drop based on age
+ *
+ * Enable or disable Tx packet drop based on packet age. When enabled, packet will be dropped
+ * if it is in Tx pktout queue or traffic shapers/schedulers for longer than timeout set.
+ *
+ * @param pkt        Packet handle
+ * @param tmo_ns     Timeout to age in nsec
+ *                >0 Aging drop is enabled with specified timeout.
+ *                =0 Aging drop is disabled.
+ *
+ * @see odp_pktio_capability_t::max_tx_aging_tmo_ns
+ */
+void odp_packet_aging_drop_set(odp_packet_t pkt, const uint64_t tmo_ns);
```

Comment:
const is not needed for tmo_ns (it's not a pointer).

I'd revise tmo_ns param documentation like this:
 param tmo_ns     Packet aging drop timeout in nsec. When 0, aging drop is disabled (default).
 
 The body text could say that that: !0 enables aging, 0 disables aging and aging is disabled by default.

Reply:
Ack.